packages/java/tests/spring/endpoints-latest-java/package.json

```diff
@@ -11,7 +11,6 @@
     "@vaadin/hilla-react-auth": "file:../../../../ts/react-auth",
     "@vaadin/hilla-react-crud": "file:../../../../ts/react-crud",
     "@vaadin/hilla-react-form": "file:../../../../ts/react-form",
-    "@vaadin/hilla-react-i18n": "file:../../../../ts/react-i18n",
```

Comment:
Why was it automatically removed? 🤔 Most likely because it's not specified in https://github.com/vaadin/hilla/blob/main/scripts/prepare/templates/components/react/package.json

Reply:
→ #2212